File: C++/ch1-3-BMI檢測.cpp
#include <iostream>
using namespace std;

int main() {
    float weight, height, bmi;
    cin >>weight >>height;
    height = height / 100;
    bmi = weight / (height * height);
    cout << "BMI值= " << bmi <<endl;
    if (bmi < 18.5) {
        cout <<"太輕了" <<endl;
    }
    else if (bmi < 24) {
        cout <<"正常" <<endl;
    }
    else{
         cout <<"太重了" <<endl;
    }
    return 0;
}

File: Python/ch1_3_bmi_檢測.py
w = input('請輸入體重(公斤):')
weight = int(w)
h = input('請輸入身高(公分):')
height = int(h) 
height = height / 100
bmi = weight / (height * height)
print('BMI值=', bmi)

if bmi < 18.5:
  print('太輕了')
elif bmi < 24:
  print('正常')
else:
  print('太重了')
